base/tks/src/org/dogtagpki/server/tks/servlet/TokenServlet.java

```diff
@@ -2926,14 +2921,16 @@ private void processComputeSessionKeysSCP03(HttpServletRequest req, HttpServletR
                     keyNickName, // KeyNickName
                     keySet, // TKSKeyset
                     log_string_from_keyInfo(xkeyInfo), // KeyInfo_KeyVersion
-            };
-            auditMessage = CMS.getLogMessage(AuditEvent.COMPUTE_SESSION_KEY_REQUEST_PROCESSED_SUCCESS,
-                    logParams);
+                    null,
+                    null
```

Comment:
The NistSP800_108KdfOnKeyVersion and NistSP800_108KdfUseCuidAsKdd were not provided by the original code. Do we want to add the values here, or remove these attributes altogether from the audit log?